src/components/Login/LoginForm.vue

```diff
@@ -112,6 +113,9 @@ export default {
     isSupported() {
       return (item) => !!(this.system_info && this.system_info.adapter?.supported?.includes(item.support))
     },
+    isSingleLoginMethod() {
+      return this.getTabs.length === 1
+    },
```

Comment:
Could there be an issue if the only available tab is for example OTP? Is it appropriate to auto-submit in that case? Perhaps we should check `this.system_info.adapter?.supported?` not just by count but also by evaluating the actual types available.